akka-docs/rst/java/remoting.rst

```diff
@@ -480,42 +476,57 @@ untrusted mode when incoming via the remoting layer:
   within the same JVM), you can restrict the messages on this interface by
   marking them :class:`PossiblyHarmful` so that a client cannot forge them.
 
-SSL
----
+Configuring SSL/TLS for Akka Remoting
+-------------------------------------
 
-SSL can be used as the remote transport by adding ``akka.remote.netty.ssl``
-to the ``enabled-transport`` configuration section. See a description of the settings
-in the :ref:`remote-configuration-java` section.
+SSL can be used as the remote transport by adding ``akka.remote.netty.ssl`` to the ``enabled-transport`` configuration section::
 
 An example of setting up the default Netty based SSL driver as default::
 
   akka {
     remote {
       enabled-transports = [akka.remote.netty.ssl]
+    }
+  }
+
+Next the actual SSL/TLS parameters have to be configured::
 
+  akka {
+    remote {
       netty.ssl.security {
-        key-store = "mykeystore"
-        trust-store = "mytruststore"
+        key-store = "/example/path/to/mykeystore.jks"
+        trust-store = "/example/path/to/mytruststore.jks"
+        
         key-store-password = "changeme"
         key-password = "changeme"
         trust-store-password = "changeme"
+        
         protocol = "TLSv1.2"
+        
+        enabled-algorithms = [TLS_DHE_RSA_WITH_AES_128_GCM_SHA256]
+        
         random-number-generator = "AES128CounterSecureRNG"
```

Comment:
Shouldn't the default documented be SecureRandom based on what Will says and the discussions on gitter today?